### PR TITLE
Hide `AbortMigration` button

### PR DESCRIPTION
**Pull request by torchiaf**
#### PR Checklist
- Is this a multi-tenancy feature/bug?
    - [ ] Yes, the relevant RBAC changes are at:
- Do we need to backport changes to the [old Rancher UI](https://github.com/rancher/u), such as RKE1?
    - [ ] Yes, the relevant PR is at:
- Are backend engineers aware of UI changes?
    - [ ] Yes, the backend owner is:

Related Issue https://github.com/harvester/harvester/issues/6193

### Occurred changes and/or fixed issues

When users abort a VM's migration, Harvester deletes the VM's `virtualmachineinstancemigrations.kubevirt.io` but it is not updating the `virtualmachine.kubevirt.io` object.
This means the UI is not receiving a new VM resource version from WS, where the AbortMigration action is removed from the actions list.

This changes will enforce the migrate/abort buttons visibility checks by looking at the status of VM's `virtualmachineinstancemigrations.kubevirt.io`

### CODE REVIEW

File: pkg/harvester/models/kubevirt.io.virtualmachine.js [hunk 1]
```diff
@@ -170,13 +170,13 @@ export default class VirtVm extends HarvesterResource {
       },
       {
         action:  'migrateVM',
-        enabled: !!this.actions?.migrate,
+        enabled: !!this.actions?.migrate && !this.vmi?.vmimResource,
```
Comment (a110605) [inline pkg/harvester/models/kubevirt.io.virtualmachine.js:173]: The `this.vmi?.vmimResource` is an empty array before VM migration, after VM migrated the `this.vmi.vmiiResource` has steveObject (see screenshot), but it won't be clean up. That casuses the migrate button is disappered.

We should allow VM migrate to node A and then able tp migrate back to node B, right ? 

Screenshot_20240724_114819
Screenshot_20240724_120215-1


IMO, the better fix is from backend to update action list and frontend just honors the action list to show corresponding action button.

**Comment by w13915984028** [comment]
Please hold this PR, I will try to fix it on backend as well. thanks.

**Comment by w13915984028** [comment]
The PR https://github.com/harvester/harvester/pull/6225 should solve this bug from the backend.

@torchiaf @a110605 thanks.

**Comment by torchiaf** [comment]
Superseded by https://github.com/harvester/harvester/pull/6225

**Pull request closed without merging**